engine/table/src/main/java/io/deephaven/engine/table/impl/BaseTable.java

```diff
@@ -347,8 +347,6 @@ public enum CopyAttributeOperation {
                 CopyAttributeOperation.Flatten,
                 CopyAttributeOperation.PartitionBy,
                 CopyAttributeOperation.Preview,
-                CopyAttributeOperation.View, // and Select, if added
```

Comment:
Let's add "breadcrumb" comments to the blocks for attributes that we *do* (conditionally) pass through select/update/view/updateView/lazyUpdate.